ql/src/test/results/clientpositive/llap/dynamic_semijoin_reduction_2.q.out

```diff
@@ -183,16 +145,8 @@ STAGE PLANS:
                             sort order: 
                             Statistics: Num rows: 1 Data size: 516 Basic stats: COMPLETE Column stats: NONE
                             value expressions: _col0 (type: decimal(34,16)), _col1 (type: decimal(34,16)), _col2 (type: tinyint), _col3 (type: tinyint), _col4 (type: bigint), _col5 (type: bigint), _col6 (type: binary)
-            Execution mode: vectorized, llap
-            LLAP IO: all inputs
-        Map 7 
-            Map Operator Tree:
-                TableScan
-                  alias: tt2
-                  filterExpr: (timestamp_col_18 is not null and decimal1911_col_16 is not null and decimal1911_col_16 BETWEEN DynamicValue(RS_13_tt1_decimal2612_col_77_min) AND DynamicValue(RS_13_tt1_decimal2612_col_77_max) and in_bloom_filter(decimal1911_col_16, DynamicValue(RS_13_tt1_decimal2612_col_77_bloom_filter))) (type: boolean)
-                  Statistics: Num rows: 1 Data size: 152 Basic stats: COMPLETE Column stats: NONE
                   Filter Operator
-                    predicate: (timestamp_col_18 is not null and decimal1911_col_16 is not null and decimal1911_col_16 BETWEEN DynamicValue(RS_13_tt1_decimal2612_col_77_min) AND DynamicValue(RS_13_tt1_decimal2612_col_77_max) and in_bloom_filter(decimal1911_col_16, DynamicValue(RS_13_tt1_decimal2612_col_77_bloom_filter))) (type: boolean)
```

Comment:
* old Map 5 was a non-sj filtered table read
* old Map 7 was an sj filtered scan

the old Map 5 and old Map 7 is now merged into the new Map 5 vertex - which is not SJ filtered because we already scan the table.
I also think that it might be beneficial to retain the SJ filter condition in cases like this and start prefiltering at this point ; I've opened HIVE-24376 to do that a few weeks ago - is that what you meaned?